src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_load_store_emitters.hpp

```diff
@@ -46,13 +53,21 @@ class jit_store_emitter : public jit_emitter {
 private:
     template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
     void emit_isa(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const;
+    template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+    void store_qbyte(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const;
+    template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+    void store_dbyte(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const;
+    template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+    void store_byte(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const;
     size_t get_aux_gprs_count() const override;
+    size_t get_aux_vecs_count() const override;
 
     std::string name_;
     int store_num_;  // the element number to store
     int byte_offset_;
     ov::element::Type src_prc_;
     ov::element::Type dst_prc_;
+    bool is_saturated_;  // true: saturated; false: truncated
```

Comment:
x64 has enum to cover `saturation` and `truncation`:
```
// Arithmetic modes for data type conversion in store_emitter
enum arithmetic_mode {
    saturation,
    truncation
};
```
I suggest to align the both archs

Reply:
Applied. Thanks Alexandra!